tests/unit/test_openapi/test_schema.py

```diff
@@ -461,13 +461,17 @@ def test_schema_generation_with_ellipsis() -> None:
 
 def test_schema_tuple_with_union() -> None:
     schema = get_schema_for_field_definition(FieldDefinition.from_annotation(Tuple[int, Union[int, str]]))
-    assert isinstance(schema.items, Schema)
-    assert schema.items.one_of == [
+    assert schema.prefix_items == [
         Schema(type=OpenAPIType.INTEGER),
         Schema(one_of=[Schema(type=OpenAPIType.INTEGER), Schema(type=OpenAPIType.STRING)]),
     ]
 
 
+def test_schema_tuple() -> None:
+    schema = get_schema_for_field_definition(FieldDefinition.from_annotation(Tuple[int, str]))
+    assert schema.prefix_items == [Schema(type=OpenAPIType.INTEGER), Schema(type=OpenAPIType.STRING)]
+
+
```

Comment:
Can you add a test that shows that this works if the tuple if enclose in another generic, e.g. `Optional[Tuple[int, int]]`?